pennylane_lightning/lightning_qubit/lightning_qubit2.py

```diff
@@ -194,7 +201,13 @@ class LightningQubit2(Device):
 
     _device_options = ("rng", "c_dtype", "batch_obs", "mcmc", "kernel_name", "num_burnin")
 
+    _new_API = True
     _CPP_BINARY_AVAILABLE = LQ_CPP_BINARY_AVAILABLE
+    short_name = "lightning.qubit2"
```

Comment:
Was the lack of a `short_name` breaking something in particular?

Reply:
At first I could not properly initialize a device with it's name and I thought I should add this by analogy with `lightning.qubit`. But the issue was the missing symlink `lightning_qubit2`, I think. So we can remove or keep it.